concrete/src/Workflow/Request/ApprovePageRequest.php

```diff
@@ -63,7 +63,7 @@ public function getWorkflowRequestDescriptionObject()
                     $c->getCollectionName(), $comments, $link));
                 $d->setDescription(t("Version %s of Page <a target=\"_blank\" href=\"%s\">%s</a> submitted for Approval.", $this->cvID, $link,
                     $c->getCollectionName()));
-                $d->setInContextDescription(t("Page Version %s Submitted for Approval.", $this->cvID));
+                $d->setInContextDescription(t("Page %s Version %s Submitted for Approval.", $c->getCollectionName(), $this->cvID));
```

Comment:
What about adding numerical positions to string placeholders (eg. `Page %1$s Version %2$s Submitted for Approval.`)?
That would allow translators to change the order of the `%s`...
Furthermore, I'd add a comment to the translatable string to describe the parameters, so that translators will know what `%1$s` and `%2$s` are:
```php
t(/*i18n: %1$s is a page name, %2$s is the ID of a page version */"Page %1$s Version %2$s Submitted for Approval.", ...)
```